x/slashing/codec.go

```diff
@@ -9,4 +9,4 @@ func RegisterCodec(cdc *codec.Codec) {
 	cdc.RegisterConcrete(MsgUnjail{}, "cosmos-sdk/MsgUnjail", nil)
 }
 
-var cdcEmpty = codec.New()
+var moduleCdc = codec.New()
```

Comment:
Shouldn't this be registered?

Reply:
good point I'll register it in an `init` here